src/app/product/product.component.html

```diff
@@ -0,0 +1,44 @@
+
+<div class="container" *ngFor="let product of p">
```

Comment:
do not iterate through products here. just pass the product as an input to this component not the products array,